plugins/inputs/procstat/procstat.go

```diff
@@ -572,4 +572,7 @@ func init() {
 	inputs.Add("procstat", func() telegraf.Input {
 		return &Procstat{}
 	})
+	inputs.Add("t128-procstat", func() telegraf.Input {
```

Comment:
Our processors are all named with `t128_`, not `t128-`

- [ ] consistent t128 prefix